src/battle_animation.cpp

```diff
@@ -250,18 +233,13 @@ BattleAnimationChara::~BattleAnimationChara() {
 	Graphics::RemoveDrawable(this);
 }
 void BattleAnimationChara::Draw() {
-	//If animation is targeted on the screen
-	if (animation.scope == RPG::Animation::Scope_screen) {
-		DrawAt(SCREEN_TARGET_WIDTH / 2, SCREEN_TARGET_HEIGHT / 2);
-		return;
```

Comment:
This is what happens when you use a diff tool you are not familiar with while rebasing >.<